packages/contracts/deploy/30_plugins/00_plugin-setups/40_placeholder_setup.ts

```diff
@@ -17,7 +18,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   const {network} = hre;
 
-  hre.placeholderBuildCIDPath = await uploadToIPFS(
+  (hre as EHRE).placeholderBuildCIDPath = await uploadToIPFS(
```

Comment:
change it on line 8 that the type of `hre` is  `EHRE`. Then you don't need to typecast

Reply:
Fixed c6f4987dadef3e215fd679d8b32ddf3df715ab8c